x-pack/test/functional/page_objects/graph_page.ts

```diff
@@ -180,13 +177,20 @@ export function GraphPageProvider({ getService, getPageObjects }: FtrProviderCon
 
     async newGraph() {
       log.debug('Click New Workspace');
-      await testSubjects.click('graphNewButton');
-      await PageObjects.common.sleep(1000);
+      await retry.try(async () => {
+        await testSubjects.click('graphNewButton');
+        await testSubjects.existOrFail('confirmModal', { timeout: 3000 });
+      });
       await PageObjects.common.clickConfirmOnModal();
+      // wait for route change to complete
+      await PageObjects.common.sleep(3000);
```

Comment:
Can we wait for any DOM changes to make it more robust?

Reply:
Makes sense, I changed that to wait for the guidance panel instead